docs/servicenow.md

```diff
@@ -0,0 +1,143 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+ with a ServiceNow object. A list of the ServiceNow fields that have been
```

Comment:
Why is every line other than the first one padded with a space? Is there some kind of alternative markdown standard I don't know about?